mp_cnn/trainers/trainer.py

```diff
@@ -10,10 +10,15 @@ def __init__(self, model, train_loader, trainer_config, train_evaluator, test_ev
         self.train_loader = train_loader
         self.batch_size = trainer_config['batch_size']
         self.log_interval = trainer_config['log_interval']
+        self.dev_log_interval = trainer_config['dev_log_interval']
```

Comment:
why do we need a different log_interval for each dataset, can the `self.log_interval` be used from the line above?

Reply:
`self.log_interval` is the interval for each regular log and `dev_log_interval` is the interval between two validations.
In your MP-CNN code, you do the evaluation every epoch. I think it could be adjusted since sometimes an epoch is too long such as for TrecQA. Then I set a small dev_log_interval so that I get more results. This is also done in SM-CNN code.